### DISCUSSION TITLE
chat gpt

### DISCUSSION THREAD

**Discussion by 443805** [community/community#59790] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why does GpT chat not work in Egypt even though we have heard that it is a great thing in dealing with artificial intelligence

**Comment by Omrigotlieb** [comment]
Try using VPN. 
This is how they do for the ban in Italy:
https://www.addictivetips.com/vpn/how-to-bypass-the-chatgpt-ban-in-italy/

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @443805! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by 443805** [reply]
كفرالدوار الثانوية بنات

**Comment by ManosPerpirakes** [comment]
If chatGPT won't work, you can also try other ai tools such as the new bing and google bard, which are similar to it. I hope this helped.
